updatecli/weekly.d/jenkinscontroller-agents.yaml

```
@@ -134,45 +140,25 @@ targets:
       file: hieradata/common.yaml
       key: "profile::jenkinscontroller::jcasc.agent_images.azure_vms_gallery_image.version"
     scmid: default
-  setInboundRubyContainerImage:
-    sourceid: getLatestInboundRubyContainerImage
-    name: "Bump ci.jenkins.io container agent image jenkinsciinfra/inbound-agent-ruby"
-    kind: yaml
-    spec:
-      file: hieradata/common.yaml
-      key: "profile::jenkinscontroller::jcasc.agent_images.container_images.jnlp-ruby"
-    transformers:
-      - addprefix: "jenkinsciinfra/inbound-agent-ruby@sha256:"
-    scmid: default
-  setInboundMaven8ContainerImage:
-    sourceid: getLatestInboundMaven8ContainerImage
-    name: "Bump ci.jenkins.io container agent image jenkinsciinfra/inbound-agent-maven (JDK8)"
-    kind: yaml
-    spec:
-      file: hieradata/common.yaml
-      key: "profile::jenkinscontroller::jcasc.agent_images.container_images.jnlp-maven-8"
-    transformers:
-      - addprefix: "jenkinsciinfra/inbound-agent-maven@sha256:"
-    scmid: default
-  setInboundMaven11ContainerImage:
-    sourceid: getLatestInboundMaven11ContainerImage
-    name: "Bump ci.jenkins.io container agent image jenkinsciinfra/inbound-agent-maven (JDK11)"
+  setInboundAllInOneContainerImage:
+    sourceid: getLatestInboundAllInOneContainerImage
+    name: "Bump ci.jenkins.io container agent image jenkinsciinfra/jenkins-agent-ubuntu-20.04 (AllInOne)"
```

```suggestion
    name: "Bump container agent image jenkinsciinfra/jenkins-agent-ubuntu-20.04 (AllInOne)"
```

=> the value is in `common.yaml` because it's shared across all puppet-managed controllers